integration_tests/wasm/dart_test.yaml

```diff
@@ -1,5 +1,9 @@
-platforms: [chrome, firefox]
-# Node doesn't work because the version available in the current Ubuntu GitHub runners is too
-# old to support WASM+GC, which would be required to run Dart tests.
-#platforms: [chrome, firefox, node]
-compilers: [dart2wasm]
+platforms:
+# TODO(https://github.com/dart-lang/test/issues/1790) - unskip
+# - chrome
```

Comment:
The linked issue is two years old and the link to the CI run in the issue is dead. Do we know if it's still relevant?

There shouldn't be any failures on Chrome. There only a few bugs in `async*` but other than that it work, and if it doesn't we should fix it instead.